### PR TITLE
Introduce basic commands for controlling CoFHWorld features

### PR DESCRIPTION
**Pull request by dizzyd**
Introduces basic structure for adding runtime control over CoFHWorld generation. Supported commands are:

> /cofh feature - prints list of available commands
> /cofh feature list - prints list of available features which CoFHWorld is generating
> /cofh feature reload - forces a reload of the CoFHWorld configs

I'm working on the ability to dynamically generate a feature on demand in front of the user, but it will have to be a separate PR as it's a significant set of changes. :/

### CODE REVIEW

**Comment by KingLemming** [comment]
So the only problem with this is that it completely changes the dep structure - right now, World doesn't depend on Core and that was very intentional.

World probably needs its own command, and if that just means copy/pasting how CoFHCore does it, so be it.

**Comment by dizzyd** [comment]
You got it, boss. Will refactor and update.

**Comment by covers1624** [comment]
Forge now has CommandTreeBase, i recommend using that.

**Comment by covers1624** [comment]
@dizzyd Are you on the twitch chat or IRC at all by any chance?

**Comment by KingLemming** [comment]
oh bleh...I guess we need to refactor CoFHCore a bit for CommandTreeBase now?

**Comment by dizzyd** [comment]
@covers1624 - I'm dizzyd on esper.net.

**Comment by dizzyd** [comment]
@KingLemming @covers1624 - I've refactored this PR, per your guidance and requests. Thoughts?

File: src/main/resources/assets/cofhworld/lang/en_US.lang [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+reload.usage=Reloads CoFHWorld configuration
+reload.successful=Configuration reloaded.
+reload.failed=Failed to reload config; check your logs.
+list.usage=Lists all CoFHWorld features
+list=Available CoFHWorld features: %s
```
Comment (covers1624) [inline src/main/resources/assets/cofhworld/lang/en_US.lang]: Localizations are global, so make sure all of our stuff is prefixed like core does. That way we avoid collisions with other mods.

**Review by covers1624 — CHANGES_REQUESTED** [review]
Looks mostly good, Just some minor localization changes. I feel that the command is a little lacking as it doesn't provide much help to the user, Perhaps something like what i did for EnderStorage's commands would be nice, but its up to lemming if it should be improved more.

**Comment by dizzyd** [comment]
I made all keys globally unique with prefixing, @covers1624

**Review by covers1624 — APPROVED** [review]

**Pull request merged**